src/MPS.jl

```diff
@@ -540,20 +541,38 @@ function mixed_canonize!(tn::AbstractMPO, orthog_center)
     return tn
 end
 
+LinearAlgebra.normalize(ψ::AbstractMPO, site) = normalize!(copy(ψ), site)
+
 LinearAlgebra.normalize!(ψ::AbstractMPO; kwargs...) = normalize!(form(ψ), ψ; kwargs...)
+LinearAlgebra.normalize!(ψ::AbstractMPO, site) = normalize!(form(ψ), ψ; at=site)
```

Comment:
use keyword dispatching with `@kwmethod` or put do a specialized version for `Base.AbstractVecOrTuple{T}` (check `length(bond) == 2`). It could also work for the `BondMixedCanonical` form.